.github/workflows/reusable_publish_docker.yaml

```diff
@@ -0,0 +1,169 @@
+# This is a reusable workflow used by master and release CI
+on:
+  workflow_call:
+
+permissions:
+  id-token: write
+  contents: read
+  
+env:
+  AWS_S3_REGION: eu-west-1
+  
+jobs:
+  publish-docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+    
+      - name: Download images tarballs
+        uses: actions/download-artifact@v3
+        with:
+          name: falco-images
+          path: /tmp
+      
+      - name: Load all images
+        run: |
+          for img in /tmp/falco-images/falco-*.tar; do docker load --input $img; done
+    
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USER }}
+          password: ${{ secrets.DOCKERHUB_SECRET }}
+      
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          role-to-assume: "arn:aws:iam::292999226676:role/github_actions-falco-ecr"
+          aws-region: ${{ env.AWS_S3_REGION }}
+          
+      - name: Login to Amazon ECR
+        id: login-ecr-public
+        uses: aws-actions/amazon-ecr-login@2f9f10ea3fa2eed41ac443fee8bfbd059af2d0a4 # v1.6.0
+        with:
+          registry-type: public    
+          
+      - name: Login to Amazon ECR Public
+        run: |
+          aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws/falcosecurity
+          
+      - name: Create and push no-driver manifest
+        uses: Noelware/docker-manifest-action@master
```

Comment:
`@master` without the commit hash is a bit risky. I'm also sure that it's there for a reason, since I see that fixes have been pushed to that repo after the latest release. Can you add the hash like you did in `login-ecr-public`?

Reply:
Since this is the first occurrence in falcosecurity org that this action is used (https://github.com/search?q=org%3Afalcosecurity+%22docker-manifest-action%22&type=code) i think i just copy pasted from the README :D 
I am going to use latest tag :)

Reply:
Done!